src/service/matchers/lineage_matcher.py

```diff
@@ -102,8 +103,7 @@ def generate_match_process(
         token - the user's token.
```

Comment:
token param is not used in this fun?

Reply:
Not for this matcher, but the minhash matcher (which implements the same method) does need it